etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataOrchestrator.java

```diff
@@ -375,7 +373,7 @@ PartnerMetadataStatus ourStatusFromReportStreamStatus(String rsStatus) {
     }
 
     private boolean metadataIsStale(PartnerMetadata partnerMetadata) {
-        return partnerMetadata.receiver() == null
+        return partnerMetadata.receivingFacilityDetails().namespace() == null
```

Comment:
I think it would be better to check for `universalId` instead of `namespace`, as the id is always required

Reply:
yes